SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -701,8 +701,10 @@ void CMeshSolver::LoadRestart(CGeometry **geometry, CSolver ***solver, CConfig *
       for (iDim = 0; iDim < nDim; iDim++){
         /*--- Update the coordinates of the mesh ---*/
         curr_coord = Restart_Data[index+iDim];
-        /// TODO: "Double" deformation if this is set here?
-//        geometry[MESH_0]->node[iPoint_Local]->SetCoord(iDim, curr_coord);
+        /// TODO: "Double deformation" in multizone adjoint if this is set here?
+        ///       In any case it should not be needed as deformation is called before other solvers
+        ///geometry[MESH_0]->node[iPoint_Local]->SetCoord(iDim, curr_coord);
+
```

Comment:
For some reason if the coordinates are set here the multizone adjoint does not work properly, it seems to use a grid with the displacements applied twice.
It does not seem to be a problem to leave this out since the mesh deformation is called before the fluid solvers, the grid velocities are computed based on displacement, and I am now initialising the solution of the linear system (i.e. it will converge in 0 iterations when restarting).